tests/druid_tests.py

```diff
@@ -125,6 +125,13 @@ def get_cluster(self, PyDruid):
             .first()
         )
         if cluster:
+            for datasource in (
+                db.session.query(DruidDatasource)
+                .filter_by(cluster_name="test_cluster")
```

Comment:
Does the `cluster` object have the `cluster_name` attribute? That may be a little cleaner/less error prone.